Sources/SpeziContact/Models/Contact.swift

```diff
@@ -11,6 +11,35 @@ import SwiftUI
 
 
 /// Encodes the contact information.
+///
+/// The following example demonstrates the usage of the `Contact`.
+/// ```swift
+/// Contact(
```

Comment:
It would be great to showcase the init of the `Contact` in a slightly bigger example, right now you just showcase the initializer